static/app/components/sidebar/usePerformanceOnboardingDocs.spec.tsx

```diff
@@ -98,8 +98,9 @@ describe('usePerformanceOnboardingDocs', function () {
       });
     });
 
-    const {result} = reactHooks.renderHook(() => usePerformanceOnboardingDocs(project), {
+    const {result} = reactHooks.renderHook(usePerformanceOnboardingDocs, {
       wrapper,
+      initialProps: project,
```

Comment:
What does this look like when it takes multiple arguments?

Reply:
TL/DR: 
```typescript
    const {result} = reactHooks.renderHook(
      (args: Parameters<typeof useLocalStorageState>) => useLocalStorageState(args[0], args[1]),
      {initialProps: ['key', 'default value']}
    );
```

---

Should start by pointing out that `initialProps` works like props in a react component: it should be one Props object with a bunch of fields, like:
```typescript
type Props = {
  project: Project;
};
function MyButton(props: Props) { ... }
```

But, we don't always write our hooks like that :(. For example, this `usePerformanceOnboardingDocs` is skipping that extra object wrapper and is implemented like this:
```typescript
type Props = Project;
function usePerformanceOnboardingDocs(project: Props) { ... }
```

---

Ok, so to the question: if the hook takes two or more arguments we have to wrap those two values into one container, because `initialProps` expects a single value, ideally an object. 

If we have this hook which takes two args:
```typescript
type FirstProp = Project;
type SecondProp = Org;
function useMyHook(project: Project, org: Org) { ... }
```

The test setup then has to turn those two values into one, and also manage the types so that things make sense.
I implemented it with these three aspects:
- We will get the types with `Parameters<typeof useMyButton>` which returns the tuple: `[Project, Org]`
- `initialProps` only accepts one value, but we can put our two values into an array which maps to what `Parameters<>` return: `initialProps: [myJSProject, myOrg],`
- We will wrap our hook in a fat-arrow method to marshal the prop value (which is an array with two values) into their spots in the method signature: `(args) => useMyHook(args[0], args[1])`